client/src/components/SessionManagerField/SessionManagerField.js

```diff
@@ -0,0 +1,17 @@
+/* global window */
+import React from 'react';
+import LoginSession from '../LoginSession/LoginSession';
+
+function SessionManagerField(props) {
```

Comment:
Can we add some proptypes to this component?

On side note, I tend to prefer deconstruction in these scenerios, but that's more of a personal choice.
```suggestion
function SessionManagerField({loginSessions}) {
```